tests/kolainst/upgrades/upgrade-rpmostree

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+## kola:
+##   # This test reboots a lot, generates container images, etc.
+##   timeoutMin: 40
+##   minMemory: 2048
+##   tags: needs-internet
+##   description: Start old build, upgrade rpm-ostree to new version,
+##     do client-side layering tests, then upgrade to another new update.
```

Comment:
Sorry, can you expand on https://github.com/coreos/rpm-ostree/pull/4784#issuecomment-1907188449 ? Isn't https://github.com/fedora-silverblue/issue-tracker/issues/523 covered by your first test where we do an `override remove` on the old version and then upgrade to the new version and then upgrade again from the new version to another version (with the same cached RPM content)?

Reply:
Yes, I meant to add there like that, but then I thought if we add such tests in kolainst that can cover more tests instead of just remove, no need to change in `e2e-upgrades`.

But agree with you that it would be better to add the tests to `e2e-upgrades` instead of kolainst.